modules/integration/tests-common/admin-clients/src/main/java/org/wso2/bps/integration/common/clients/bpmn/ActivitiRestClient.java

```diff
@@ -0,0 +1,684 @@
+package org.wso2.bps.integration.common.clients.bpmn;
+
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpHost;
+import org.apache.http.HttpResponse;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.entity.mime.MultipartEntityBuilder;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.apache.commons.logging.Log;
+
+import java.io.File;
+
+public class ActivitiRestClient {
+
+	private final static String username = "admin";
+	private final static String password = "admin";
+	private final static String userClaim = "paul";
+	private final static String userDelegate = "will";
```

Comment:
Removed all hardcoded urls and ports. Created a constructor to pass the host and the post from the automation context to create the service url string. Fixed in the latest commit.